### ISSUE TITLE
Youtube scrobbler bugs out with playlists

### ISSUE THREAD

**Issue by TheAsuro** [david-sabata/web-scrobbler#601]
The scrobbler activates only after finishing the first song and seems to be behind by 1 song in the playlist. It will read the song correcly after refreshing the page, but after loading the next song it will stay on the old one again.
This only happens in youtube playlists, everything else is fine. I'm using Chrome Version 42.0.2311.135 m

Edit: This was already reported in #423

**Comment by gregcarniel** [comment]
I have the same problem. 

Chrome Version 43.0.2357.81
Yosemite OS

**Comment by schleumer** [comment]
Same here, i'm not even using playlist. I think it's because the new Youtube UI doesn't refresh the page.

Chrome Version 43.0.2357.81m
Windows 8.1

**Comment by pink999** [comment]
REFRESH if you can not fix - just say so - can you do that?

**Comment by janbaykara** [comment]
I cannot replicate this at all. Youtube playlists seem to be working.

**Comment by pink999** [comment]
hang them! on wall ! thanks again! xbrix

**Comment by janbaykara** [comment]
@david-sabata I think we can close this, seems a duplicate of a solved issue.
